src/app/client/src/app/modules/workspace/components/batch-page-section/batch-page-section.component.ts

```diff
@@ -201,6 +201,25 @@ export class BatchPageSectionComponent extends WorkSpace implements OnInit, OnDe
       this.showLoader = false;
       return;
     }
+    const courseIds = _.map(this.batchList, 'courseId');
+    const searchOption = {
+      'filters': {
```

Comment:
query should be sent only **name** and **identifier** so that they are only retrived